scylla/src/transport/connection.rs

```diff
@@ -58,15 +62,14 @@ struct TaskResponse {
 pub struct ConnectionConfig {
     pub compression: Option<Compression>,
     pub tcp_nodelay: bool,
+    pub use_tls: bool,
+    pub tls_certificate_path: Option<String>,
```

Comment:
I know that these two attributes come from a comment, but perhaps it's enough to just leave `tls_certificate_path`? It looks like the certificate will be present if and only if `use_tls` is true, so since the certificate type is already `Option<String>`, we can express not using tls as having no certificate.

Reply:
(Unless we expect to have other ways of configuring TLS in the future than via `tls_certificate_path` then both attributes make sense.)